lib/hoe/signing.rb

```diff
@@ -70,7 +70,8 @@ def set_up_signing # :nodoc:
     end
 
     if signing_key and cert_chain then
-      spec.signing_key = OpenSSL::PKey::RSA.new File.read signing_key
+      passphrase = ENV['GEM_PRIVATE_KEY_PASSPHRASE']
+      spec.signing_key = OpenSSL::PKey::RSA.new(File.read(signing_key), passphrase)
```

Comment:
Mmmmm... what if the signing key isn't signed? Is `nil` OK?

Reply:
It _seems_ fine with it... But I'm not much of a crypto guy.

Reply:
I verified a second time that all 4 cases work as expected:
1. plaintext key with arg `nil`: loads the key
1. plaintext key with password arg: Ignores the password and loads the key
1. encrypted key with arg `nil`: Shows password prompt `Enter PEM pass phrase:` and loads the key if password matches
1. encrypted key with password arg: No password prompt and loads the key if password matches

Reply:
awesome. thanks... this needs some doco so I'll add that next.